server/src/main/java/org/jetbrains/bsp/bazel/server/bloop/ScalaInterop.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.bsp.bazel.server.bloop
+
+import scala.Option
+import scala.Some
+import scala.collection.immutable.List
+import scala.collection.immutable.`List$`
+import scala.jdk.javaapi.CollectionConverters.asScala
+import java.nio.file.Path
+
+object ScalaInterop {
+    fun <T> emptyList(): List<T> = `List$`.`MODULE$`.empty()
+
+    fun <T> emptyOption(): Option<T> = scala.Option.empty<T>()
+
+    fun <T> T?.toOption(): Option<T> =
+        let { Some.apply(it) } ?: Option.empty()
```

Comment:
it maps null to `Some(null)`

```suggestion
        Option.`when`(this != null) { this }
```